robottelo/cli/hammer.py

```diff
@@ -38,11 +40,37 @@ def _csv_reader(output):
             yield row
 
 
+def _normalize_header(header):
+    """Replace empty spaces with '-' and lower all chars"""
+    return header.replace(' ', '-').lower()
+
+
+def parse_json(stdout):
+    """Parse JSON output from Hammer CLI and convert it to python dictionary
+    while normalizing keys."""
+    parsed = json.loads(stdout)
+    return _normalize_obj(parsed)
+
+
+def _normalize_obj(obj):
+    """Normalize all dict's keys replacing empty spaces with "-" and lowering
+    chars"""
+    if isinstance(obj, dict):
+        return {_normalize_header(k): _normalize_obj(v) for k, v in
+            obj.items()}
+    elif isinstance(obj, list):
+        return [_normalize_obj(v) for v in obj]
+    # doing this to conform to csv parser
+    elif isinstance(obj, int) and not isinstance(obj, bool):
+        return str(obj)
```

Comment:
Since the idea is to evolve and use JSON than CSV let's leverage the JSON typing and use int instead of string. I think this can be dropped and tests adjusted.

This will also benefit tests that will eventually need to convert back to int to check for `==`, `>`, `<`, `>=`, `<=`, ...